code/modules/mapfluff/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -138,12 +150,18 @@ GLOBAL_VAR_INIT(hhMysteryRoomNumber, rand(1, 999999))
 		return TRUE
 	return FALSE
 
-/obj/item/hilbertshotel/proc/sendToNewRoom(roomNumber, mob/user)
+/obj/item/hilbertshotel/proc/sendToNewRoom(roomNumber, mob/user, chosen_room) //SKYRAT EDIT ADDITION - GHOST HOTEL UPDATE
```

Comment:
You need to mention what that change you did is, so it's not a pain to figure out when it gets changed from upstream.

Reply:
Done